packages/esm-patient-search-app/src/compact-patient-search/compact-patient-search.component.tsx

```diff
@@ -24,14 +23,17 @@ const CompactPatientSearchComponent: React.FC<CompactPatientSearchProps> = ({
   shouldNavigateToPatientSearchPage,
 }) => {
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
-  const showSearchResults = useMemo(() => !!searchTerm.trim(), [searchTerm]);
+  const debouncedSearchTerm = useDebounce(searchTerm);
+  const hasSearchTerm = useMemo(() => Boolean(debouncedSearchTerm.trim()), [debouncedSearchTerm]);
```

Comment:
Do we need the `useMemo()` here? It seems like `Boolean(debouncedSearchTerm.trim()` is a fast operation...

Reply:
No, we don't. Good catch.